eng/tools/generator/cmd/template/templateCmd.go

```diff
@@ -85,6 +85,7 @@ type Flags struct {
 	ReleaseDate   string
 	PackageConfig string
 	GoVersion     string
+	Version       string
```

Comment:
What is this `Version` property used for? Could we have some comment on these properties?

Reply:
And this new property is not added to the `BindFlags` above, is this an issue?

Reply:
Changed to PackageVersion.